contracts/BurnAuction.sol

```diff
@@ -1,6 +1,6 @@
 pragma solidity ^0.5.15;
 
-import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+import {SafeMath} from "./SafeMath.sol";
```

Comment:
Wonder why we change to copy-pasting SafeMath.sol to the codebase. Does the import not work somehow?

Reply:
actually its this errors
https://forum.openzeppelin.com/t/import-openzeppelin-contracts-on-windows-could-not-find-import-from-any-sources-error/2656/3
I can revert back to old code without copy-pasted library since docgen didn't gave expected results.
let me know if thats fine

Reply:
I see. No worries, both way works, I'm just asking.